app/common/containers/pages/SignInPage/redux.js

```diff
@@ -19,15 +19,11 @@ dispatch(createSessionToken({
     const { message } = action.payload.response.error;
     if (message === 'User blocked.') {
       throw new SubmissionError({
-        email: { user_blocked: true },
+        email: { accountPasswordMismatch: true },
       });
     } else if (message === 'Identity, password combination is wrong.') {
       throw new SubmissionError({
-        password: { passwordMismatch: true },
-      });
-    } else if (message === 'User not found.') {
-      throw new SubmissionError({
-        email: { identityMismatch: true },
+        email: { accountPasswordMismatch: true },
```

Comment:
What is the difference between accountPasswordMismatch and passwordMismatch?

Reply:
AccountPasswordMismatch goes for not valid password or email. passwordMismatch - is only for password. So there is different text for them.

Reply:
IMO there is too many names for this occurrence  it makes this situation confusing. 
Message says: 'identity', error is 'account' and in reality it is about email. Why don't make message 'Email or password is wrong' and error 'EmalOrPasswordMismatch' for example.